src/modules/pos/validator/components/ValidatorProfile/DetailsView.js

```diff
@@ -83,37 +86,44 @@ const DetailsView = ({ data, isMyProfile }) => {
         <h1 className={styles.heading}>{t('Details')}</h1>
       </BoxHeader>
       <BoxContent className={`${styles.details} details`}>
-        {displayList.map(({ icon, label, value, onEdit }) => (
+        {displayList.map(({ icon, label, value, onView, onEdit }) => (
           <div key={label} className={`${grid.row} ${styles.itemContainer}`}>
             <Icon name={icon} className={styles.icon} />
             <div className={`${grid.col} ${styles.item}`}>
               <div className={`${styles.title} ${theme}`}>
                 <span>{label} </span>
-                {onEdit &&
-                  typeof onEdit === 'function' &&
-                  (hasChangeCommission ? (
-                    <Tooltip
-                      className={classNames(styles.editDisabledIcon, styles.editBtn)}
-                      tooltipClassName={`${styles.tooltipClassNameProp}`}
-                      size="maxContent"
-                      position="bottom"
-                      content={<Icon name="editDisabled" />}
-                    >
-                      <p>
-                        {t(
-                          'You have to wait for your current commission change to finalize before you can edit again.'
-                        )}
-                      </p>
-                    </Tooltip>
-                  ) : (
-                    <button
-                      onClick={onEdit}
-                      className={styles.editBtn}
-                      disabled={hasChangeCommission}
-                    >
-                      <Icon name="editActiveIcon" />
```

Comment:
Pen icon has become distorted in this PR, somehow padding is gone. These icons needs to be align in middle and same size
![image](https://github.com/LiskHQ/lisk-desktop/assets/8784876/640ad656-200b-4ab2-90dc-d557e0b55cce)

Reply:
Fixed